pkg/operator/workload/workload_openshiftapiserver_v311_00_sync_test.go

```diff
@@ -219,3 +237,155 @@ func TestPreconditionFulfilled(t *testing.T) {
 		})
 	}
 }
+
+func TestCapabilities(t *testing.T) {
+	testCases := []struct {
+		name                    string
+		knownCapabilities       []configv1.ClusterVersionCapability
+		enabledCapabilities     []configv1.ClusterVersionCapability
+		expectedPerGroupOptions []openshiftcontrolplanev1.PerGroupOptions
```

Comment:
would it make sense to validate the whole config and not only `PerGroupOptions` ?
assuming that other fields remain unchanged, this shouldn't require much more effort

Reply:
That would require to construct a full config to compare it with. Also, diff.ObjectDiff over `config.APIServers.PerGroupOptions` is easier to read.

Reply:
Done